package.json

```diff
@@ -87,11 +87,14 @@
     "swarm-icons": "1.3.259",
     "tough-cookie": "2.3.2"
   },
+  "peerDependencies": {
+    "mwp-cli": "^1.1.268 || ^2.0.0"
+  },
```

Comment:
this peerDep doesn't need to be here because there is no top-level package that can be installed